sources/assets/exegol/load_supported_setups.sh

```diff
@@ -1,9 +1,8 @@
-#!/bin/bash
+#!/bin/zsh
 
 # Expanding aliases. Bugfix #207
 EXEGOL_ALIASES="/opt/.exegol_aliases"
 if [[ -f "$EXEGOL_ALIASES" ]]; then
-  shopt -s expand_aliases
   source "$EXEGOL_ALIASES"
 else
   echo "Exiting, file $EXEGOL_ALIASES is missing"
```

Comment:
No, I left it on purpose. Looking at the commit you gave, the aliases are indeed loaded in the zshrc. However, the zshrc is automatically loaded from within an interactive session. It's not the case here as it is a script.
It's therefore required to explicitely source $EXEGOL_ALIASES.
Plus, I had tested without it, bqm still fails to be found.

Reply:
Then would it be possible to replace this block with a `source ~/.zshrc` ?

Reply:
This will allow the script to follow the changes made to zhsrc in the future

Reply:
I see your point, but I discourage this approach as it could introduce some side-effects in the script execution depending on what's added into zshrc. A side-effect that would go unnoticed as we've seen as there's currently no check on this file when running the pipeline. This is why the scope extension should be on a need-basis to avoid any issues.
I believe it would be a lot easier to troubleshoot a missing source aliasesfile than any funky commands/variables from a zshrc.
If you absolutely want me to source ~/.zshrc I'll do it though.

Reply:
Hmm, I see your point, we'll go as-is, and in the future, if we make any check of the zshrc content, or if we want to switch to what I described for some reason, we will. However, while I agree with you on the side-effects, there are benefits to sourcing zhsrc. One example is what we have with pyenv. Pyenv is set up in the zshrc, but not in the load_supported_setups, preventing python/pip from working correctly (because it's not in the PATH).
The example with pyenv could be encountered with other things that are set up in zshrc and not elsewhere, and sourcing zshrc would allow to not have duplicate code hanging around

Reply:
Yep, valid point, sometimes the benefits are worth taking some risks ;), and indeed it could be more interesting to switch.